src/runtime_src/core/common/xrt_graph.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020, Xilinx Inc - All rights reserved
+ * Xilinx Runtime (XRT) Experimental APIs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
I realized also on some other projects that this formatting style of Xilinx legal text is annoying because spamming it with a lot of stars prevent it to be copy-pasted and add a lot of `*` in the middle of some legal normative text...
Can someone reach out legal people so their text is not changed like that?

Reply:
For all new code you can use SPDX style license:
`/* SPDX-License-Identifier: Apache-2.0 */`

Reply:
added SPDX licence for two newly added files